### DISCUSSION TITLE
Persistent Storage Options

### DISCUSSION THREAD

**Discussion by gprossliner** [mysticaltech/terraform-hcloud-kube-hetzner#645] [Q&A] [answered]
Hi! This project looks promising. Thank you.

I was not able to find any information about how the CSI / StorageClass configuration that will be applied. 

* Is there any setup for persistent storage while provisioning a cluster?
* Is there a CSI Plugin to allocate Hetzner Volumes?
* Do you expect users to apply Longhorn for storage?

It would be nice if something about storage is mentioned in the readme.

Thank you!

**Comment by mysticaltech** [comment, marked as answer]
@gprossliner In kube.tf.example, we explain what we support, basically either Hetzner CSI or Longhorn, in that flag you can choose to enable either or both. 

Personally, I think Longhorn is a more robust option, and to configure it properly, either search this project's issues and discussions for examples, you will find quite a few, or read the [longhorn docs](https://longhorn.io/docs/).

**Reply by gprossliner** [reply]
Thank you, I will take a look.
